.github/workflows/build.yml

```diff
@@ -179,7 +179,7 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        python: ['3.6', '3.7']
+        python: ['3.8']
```

Comment:
@yongtang maybe we can test with 3.7 and 3.8? Just to be sure of the compatibility as users might still be using 3.7.